packages/core/comparison/lib/isIntentionalDifference/index.ts

```diff
@@ -44,7 +45,8 @@ const filters = [
   invalidManualSequenceNumber,
   nonMatchingManualSequenceNumber,
   offenceReasonSequenceFormat,
-  prioritiseNonFinal
+  prioritiseNonFinal,
+  bichardRaisesHo200114ForNonExactSequenceNumbers
```

Comment:
I think the rest of the items in this array are in alphabetical order

Reply:
Ahhhhh yupppp, they are - I've fixed that now!